### DISCUSSION TITLE
Itron C2SO SK9AMI7 Bidirectional meter support

### DISCUSSION THREAD

**Discussion by JoeDaddy7105** [bemasher/rtlamr#208] [Ideas]
I recently had solar installed and they swapped out my main electrical meter for a bi-directional meter. I was able to pickup the scm+ signal but it appears to only be getting the consumption counter that counts up when we use grid energy. When the solar is overproducing and sending energy back to the grid, the consumption counter plateaus and doesn't go down. I assume this means there is a separate production counter that ticks up whenever we are pushing energy back to the grid but I can't seem to find that signal. Wondering if that might be a different protocol or under a different id but I can't find anything that would make sense when I scan with other protocols.

![PXL_20210921_200829423-2](https://user-images.githubusercontent.com/24470441/134796086-00fed03c-6bd3-4c39-a86f-bd065d336ba5.jpg)

I also noticed the endpoint type picked up during scanning is 110 but the ERT type on the meter is 14. Wondering if the endpoint type would be different for the production counter but all I see is type 110 in all my readings.
![Screenshot 2021-09-26 000115](https://user-images.githubusercontent.com/24470441/134796100-3fbf6af3-cd0b-4455-a81c-58bd2eddf754.png)

**Comment by bemasher** [comment]
I have seen these meters transmit messages under multiple endpoint id's, usually sequentially. So if you see 10100005, you might also look for 10100005 or 101000004, etc.
